Bio/KEGG/Compound/__init__.py

```diff
@@ -171,16 +171,9 @@ def parse(handle):
                     enzyme = (column.strip(), "")
                 record.enzyme.append(enzyme)
         elif keyword == "PATHWAY     ":
-            if data[:5] == 'PATH:':
-                path, map, name = data.split(None, 2)
-                pathway = (path[:-1], map, name)
-                record.pathway.append(pathway)
-            else:
-                pathway = record.pathway[-1]
-                path, map, name = pathway
-                name = name + " " + data
-                pathway = path, map, name
-                record.pathway[-1] = pathway
+            map, name = data.split("  ")
+            pathway = (map, name)
```

Comment:
Imo this is bad because it breaks the API. Before the tuple was ("PATH", map, name) now it is (map, name). Any code that relies on these indexing (probably alot) will break.

Better would be pathway = ("PATH", map, name)